tests/gdx-tests-android/AndroidManifest.xml

```diff
@@ -4,7 +4,7 @@
       android:versionCode="1"
       android:versionName="1.0"
       android:installLocation="preferExternal">
-	<uses-sdk android:minSdkVersion="8" android:targetSdkVersion="19"/>
+	<uses-sdk android:minSdkVersion="9" android:targetSdkVersion="19"/>
```

Comment:
I feel like, since we don't actually know if the user is going to use support libraries, we should default to the minimum required by libgdx itself, not some library that someone may or may not use.